js/apps/admin-ui/src/realm-settings/user-profile/AttributesGroupForm.tsx

```diff
@@ -110,82 +110,63 @@ export default function AttributesGroupForm() {
         titleKey={matchingGroup ? "editGroupText" : "createGroupText"}
         divider
       />
-      <PageSection variant="light" onSubmit={form.handleSubmit(onSubmit)}>
```

Comment:
How strange the `onSubmit` handler was attached to the `PageSection`, it doesn't seem the right place.